templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -84,7 +111,7 @@
     "anyscale-data-cld-id/org_id/cloud_id/artifact_storage/username/llmforge-finetuning/meta-llama/Llama-2-70b-hf/TorchTrainer_2024-01-25_18-07-48/TorchTrainer_b3de9_00000_0_2024-01-25_18-07-48/checkpoint_000000\n",
     "```\n",
     "\n",
-    "You can follow the `Learn how to bring your own models` section under the endpoints template to serve this finetuned model with the specified storage uri."
+    "You can follow the [Learn how to bring your own models](../endpoints_v2/examples/CustomModels.ipynb#adding-a-private-model) section under the llm serving template to serve this finetuned model with the specified storage uri."
```

Comment:
I don't think this actually works since the workspace clone isn't a full clone. How about linking to https://docs.endpoints.anyscale.com/preview/examples/deploy-llms/ instead?